src/cli.rs

```diff
@@ -70,6 +134,50 @@ fn outpoint_validator(s: String) -> Result<(), String> {
     parse_outpoint(&s).map(|_| ())
 }
 
+/// Create new [clap](https://docs.rs/clap/) `App` with wallet sub-commands
+///
+/// Configuration such as name, version, author, about can be overridden. Additional sub-commands
+/// sub-commands can also be added. See [clap](https://docs.rs/clap/) for more information on how
+/// to customize an `App`.
+///
+/// # Wallet sub-commands
+///
+/// | Sub-command        | Description                                    |
+/// |--------------------|----------------------------------------------- |
+/// |  broadcast         |   Broadcasts a transaction to the network. Takes either a raw transaction or a PSBT to extract |
+/// |  bump_fee          |   Bumps the fees of an RBF transaction |
+/// |  combine_psbt      |   Combines multiple PSBTs into one |
+/// |  create_tx         |   Creates a new unsigned tranasaction |
+/// |  extract_psbt      |   Extracts a raw transaction from a PSBT |
+/// |  finalize_psbt     |   Finalizes a psbt |
+/// |  get_balance       |   Returns the current wallet balance |
+/// |  get_new_address   |   Generates a new external address |
+/// |  help              |   Prints this message or the help of the given subcommand(s) |
+/// |  list_transactions |   Lists all the incoming and outgoing transactions of the wallet |
+/// |  list_unspent      |   Lists the available spendable UTXOs |
+/// |  policies          |   Returns the available spending policies for the descriptor |
+/// |  public_descriptor |   Returns the public version of the wallet's descriptor(s) |
+/// |  repl              |   Opens an interactive shell |
+/// |  sign              |   Signs and tries to finalize a PSBT |
+/// |  sync              |   Syncs with the chosen Electrum server |
```

Comment:
Documenting this way the cli it's hard to maintain because we have the same phrases in the code and the doc. I suggest moving to structopt (uses clap under the hood) where you document a struct and the documentation is used for both the documentation itself and the suggestion in the cli

Reply:
I'm quite satisfied with `structopt`, but one limitation worth noting is that it doesn't support loading options from configuration files. It does support env variables though, and could be configured via a `.env` file, but that has its limitations too.

Reply:
@RCasatta oh nice idea, anything that will make keeping docs in sync we should do. I'll try using a `structopt` so we can see what it looks like.

Reply:
I've switched to StructOpt and removed this from docs.